docs/html/copyright.rst

```diff
@@ -0,0 +1,9 @@
+:orphan:
+
+=========
+Copyright
+=========
+
+pip and this documentation is:
+
+Copyright © 2008-2020 The pip developers (see `AUTHORS.txt <https://github.com/pypa/pip/blob/master/AUTHORS.txt>`_ file). All rights reserved.
```

Comment:
This probably shouldn't be hard-coded

Reply:
Not sure of any alternatives.

Reply:
Would it be a good idea to include `AUTHORS.txt` as part of the documentation as well? e.g. creating a `authors.rst` with `.. include:: ../AUTHORS.txt`.

Reply:
@RDIL you can declare substitutions: http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions

Reply:
I'm weary of more prominently showing a bunch of emails in our documentation. :/

@webknjaz Any specific reasons to avoid hardcoding this?

Reply:
Yet another place to update 2020 is error-prone

Reply:
Oh, it's probably unclear that I was talking about the dates range, not the authors list.

Reply:
Anything blocking this specific PR?

Reply:
Looks like all reviews are either comments or approvals. So seems you're good.